packages/functional-tests/tests/settings/password.spec.ts

```diff
@@ -22,7 +22,10 @@ test.describe('severity-1 #smoke', () => {
     );
     await page.goto(link, { waitUntil: 'load' });
     await login.setNewPassword(credentials.password);
-    expect(page.url()).toContain(settings.url);
+    // TODO: React reset PW does not currently take users to Settings. See related comment:
+    //https://github.com/mozilla/fxa/blob/24a842fccea6782783500929444bebaf90d0564b/packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx#L276-L277
```

Comment:
👍🏽

Reply:
Follow up filed and noted in code: FXA-8266